apps/coscrad-frontend/src/components/footer/footer.tsx

```diff
@@ -1,18 +1,34 @@
+import { Stack, Typography } from '@mui/material';
+import Box from '@mui/material/Box';
 import { useContext } from 'react';
 import { ConfigurableContentContext } from '../../configurable-front-matter/configurable-content-provider';
+import { Copyright } from '../copyright/copyright';
 import { COSCRADByline } from '../coscrad-byline/coscrad-byline';
 import { TenantLogo } from '../tenant-logo/tenant-logo';
 
 export const Footer = (): JSX.Element => {
-    const { copyrightHolder } = useContext(ConfigurableContentContext);
+    const { subTitle, copyrightHolder } = useContext(ConfigurableContentContext);
 
     return (
-        <footer>
-            {/* TODO: create date component so we have control over this for snapshots and tests */}
-            &copy; {new Date().getFullYear()} {copyrightHolder}
-            <TenantLogo />
-            <br />
-            <COSCRADByline />
-        </footer>
+        <Box
+            component="footer"
+            sx={{
```

Comment:
One thing about `sx` props is that they use theme variables

Reply:
That's interesting. Sounds like a powerful approach but maybe a bit opaque. Akin to "aspect oriented programming" magic on the back-end, if I'm understanding correctly.